api/c/indigo/src/indigo_reaction.cpp

```diff
@@ -383,9 +385,11 @@ CEXPORT int indigoLoadQueryReaction(int source)
 
         loader.stereochemistry_options = self.stereochemistry_options;
         loader.treat_x_as_pseudoatom = self.treat_x_as_pseudoatom;
+        loader.dearomatize_on_load = self.dearomatize_on_load;
+        loader.arom_options = self.arom_options;
 
         std::unique_ptr<IndigoQueryReaction> rxnptr = std::make_unique<IndigoQueryReaction>();
-        loader.loadQueryReaction(rxnptr->rxn);
+        loader.loadReaction(rxnptr->rxn);
```

Comment:
?

Reply:
We don't need separate loadQueryReaction. loadReaction can handle both types.